### PR TITLE
fix: button enter handling

### PR DESCRIPTION
**Pull request by noahwaldner**

### CODE REVIEW

File: packages/components/src/components/Button/Button.tsx [hunk 1]
```diff
@@ -71,8 +74,9 @@ export const Button = forwardRef<HTMLButtonElement | null, ButtonProps>(
             children,
             style,
             size = 'medium',
-            'data-test-id': dataTestId = 'fondue-button',
+            'data-test-id': dataTestId = '  fondue-button',
```
Comment (SamuelAlev) [inline packages/components/src/components/Button/Button.tsx:77]:
```suggestion
            'data-test-id': dataTestId = 'fondue-button',
```

File: packages/components/src/components/Button/Button.tsx [hunk 2]
```diff
@@ -86,7 +90,12 @@ export const Button = forwardRef<HTMLButtonElement | null, ButtonProps>(
                     textStyles({ style, ...props }),
                     iconStyles({ style, ...props }),
                     className,
+                    FOCUS_OUTLINE,
                 )}
+                onClick={onPress}
+                onKeyDown={(e: KeyboardEvent<HTMLButtonElement>) => {
+                    e.key === 'Enter' && onPress(e);
```
Comment (SamuelAlev) [inline packages/components/src/components/Button/Button.tsx:97]:
```suggestion
                onKeyDown={(event: KeyboardEvent<HTMLButtonElement>) => {
                    event.key === 'Enter' && onPress(e);
```

File: packages/components/src/components/Button/styles/buttonStyles.ts [hunk 3]
```diff
@@ -3,7 +3,7 @@
 import { sv } from '#/utilities/styleUtilities';
 
 export const buttonStyles = sv({
-    base: 'tw-group tw-border tw-box-box tw-relative tw-flex tw-flex-row tw-gap-2 tw-items-center tw-justify-center tw-cursor-pointer tw-outline-none tw-font-body tw-font-medium',
+    base: 'tw-group tw-border tw-box-box tw-relative tw-flex tw-flex-row tw-gap-2 tw-items-center tw-justify-center tw-cursor-pointer tw-font-body tw-font-medium',
```
Comment (SamuelAlev) [inline packages/components/src/components/Button/styles/buttonStyles.ts:6]: Out of scope: I think `tw-box-box` doesn't exist

Comment (noahwaldner) [reply packages/components/src/components/Button/styles/buttonStyles.ts]: removed

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**